jsapp/js/components/modalForms/translationTable.es6

```diff
@@ -67,7 +82,18 @@ export class TranslationTable extends React.Component {
         Header: t('Original string'),
         accessor: 'original',
         minWidth: 130,
-        Cell: (cellInfo) => {return cellInfo.original.original;}
+        Cell: (cellInfo) => {
+          // Disabling has no effect on this cell, but we do it to gray out the
+          // text to indicate that the label is locked
+          // TODO: Figure out what to do for the case of adding a new language
+          //       when there are locked labels. These labels should be unlocked
+          //       for the newly added languages and their translations only.
+          return (
+            <div className={cellInfo.original.isLabelLocked ? 'disabled' : ''}>
```

Comment:
Maybe make this `rt-td--disabled`, as `disabled` is quite a common name, and could be defined elsewhere (e.g. in some plugin we use and we update)?